spec/store/watch-spec.js

```diff
@@ -52,4 +56,17 @@ describe("WatchStore", () => {
       expect(store.editor.element.focus).toHaveBeenCalled();
     });
   });
+  describe("addAutocomplete", () => {
+    beforeEach(
+      waitAsync(async () => {
+        const p = await atom.packages.activatePackage("autocomplete-plus");
+        //modelled after consumeWatchEditor in lib/main.js
+        _store.enableAutocomplete(p.mainModule.provideWatchEditor());
```

Comment:
agreed

Reply:
As it stands I can't not use the global store since thats what the code inside `watchStore` uses, but I could instead just always be passing the `watchEditor` function down the chain to avoid using the global store